packages/rulesets/src/spectral/functions/xms-pageable-for-get-list-calls.ts

```diff
@@ -0,0 +1,17 @@
+const xmsPageableForGetListCalls = (swaggerObj: any, _opts: any, paths: any) => {
+  if (swaggerObj === null) {
+    return []
+  }
+  const path = paths.path || []
+
+  if (swaggerObj["x-ms-pageable"]) return []
+  else
```

Comment:
Lets also check that the definition of the extension is also specified correctly. 

The extension must have the 
nextLinkName property defined

Reply:
That is already covered in native rule